CHANGES.rst

```diff
@@ -4,7 +4,8 @@ Changes
 4.4.3 (unreleased)
 ------------------
 
-- Nothing changed yet.
+- Add support for general-purpose Python 3.x annotations / type-hints in interface definitions.
```

Comment:
Yes, it is mypy. Ideally, I'd like to have annotations that hold type information as well. I stumbled upon the case when having a type hint for an attribute declaration resulted in runtime error:
```python

class ICriterion(Interface):
    name: str = Attribute('name')
```

Even though it's not correct from mypy's perspective either (Attribute() is not an instance of string). So far I'd like to just have this combination of hints and interfaces present at the same time.